drivers/storage/portworx/util/util_test.go

```diff
@@ -252,6 +255,70 @@ func TestIsEmptyOrNilCertLocation(t *testing.T) {
 
 }
 
+func TestGetServiceTypeFromAnnotation(t *testing.T) {
```

Comment:
add test cases to update service type, since it did not work before customer will need to update service type.